CONTRIBUTING.md

```diff
@@ -47,11 +47,13 @@ First, ensure that [Yarn](https://yarnpkg.com/) and
 
 ## The demo project
 
-The demo project was initially forked from
+The [demo project](demo) was initially forked from
```

Comment:
I think we can skip now that it's a fork?

Reply:
Not sure what you meant @shcheklein but I'm about to open another PR on this doc so we can follow up there ⏳

Reply:
https://github.com/iterative/vscode-dvc/pull/1928